lib/src/conversationlist.dart

```diff
@@ -105,7 +105,7 @@ class ConversationList extends StatefulWidget {
     this.showFeedHeader,
     this.theme,
     this.themeOptions,
-    this.feedFilter = const ConversationPredicate(),
```

Comment:
Why change this? If we set `feedFilter` to `BaseConversationPredicate` we can revert this change and most of the others in this file.